packages/react-native/src/evm/wallets/connectors/embedded-wallet/embedded/helpers/wallet/encryption.ts

```diff
@@ -1,19 +1,21 @@
 import AesGcmCrypto from "react-native-aes-gcm-crypto";
-import { getRandomValues } from "../getRandomValues";
 import QuickCrypto from "react-native-quick-crypto";
+import { getRandomValues } from "../getRandomValues";
 
 const ENCRYPTION_SEPARATOR = ":";
-const KEY_ITERATION_COUNT = 5000000;
+const DEPRECATED_KEY_ITERATION_COUNT = 5_000_000;
+const CURRENT_KEY_ITERATION_COUNT = 650_000;
```

Comment:
Please, don't use this notation, devs will need to add babel plugins to be able to understand it:

https://github.com/facebook/react-native/issues/31551